site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -189,6 +193,112 @@ To prepare the manifest file, do the following:
 <iframe width="560" height="315" src="https://www.youtube.com/embed/y6fgZ4X87Lc?si=5EewLS4XA7birS25" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 <!--lint enable maximum-line-length-->
 
+## Backblaze B2
+
+Backblaze B2 is a cost-effective S3-compatible cloud storage platform
+that offers storage costs up to 5× cheaper than AWS S3. B2 Overdrive
+provides 1 TB of throughput with free egress, making it an excellent
+choice for large-scale annotation projects.
+
+Since B2 is S3-compatible, you can use it in CVAT by selecting **AWS S3**
+as the provider and configuring the B2-specific endpoint.
+
+### <a name="create-a-bucket-b2">Create a bucket</a>
+
+To create a B2 bucket, do the following:
+
+1. Create a [Backblaze account](https://www.backblaze.com/sign-up/cloud-storage) or log into an existing one.
+1. In the Backblaze console, go to **B2 Cloud Storage** > **Buckets**.
+1. Select **Create a Bucket**.
+1. Configure your bucket:
+   - **Bucket Unique Name**: Enter a globally unique name for your bucket.
+   - **Files in Bucket**: Select **Private** for secure access or **Public** for anonymous access.
+   - **Default Encryption**: (Optional) Enable server-side encryption for added security.
+   - **Object Lock**: (Optional) Enable if you need compliance features.
+1. Select **Create a Bucket**.
+
+The new bucket will appear in your buckets list.
+
+### <a name="upload-data-b2">Upload data</a>
+
+{{% alert title="Note" color="primary" %}}
+The manifest file is optional.
+{{% /alert %}}
+
+You need to upload data for annotation and optionally the `manifest.jsonl` file.
+
+1. Prepare data.
+   For more information,
+   refer on how to [prepare the dataset](#prepare-the-dataset).
+1. Open the bucket and select **Upload/Download**.
+1. Drag and drop files or folders, or select **Browse files** to upload your data.
+
+Alternatively, you can use the [Backblaze CLI](https://www.backblaze.com/docs/cloud-storage-command-line-tools)
+or any S3-compatible tool like the AWS CLI with B2 endpoints.
+
+### <a name="access-permissions-b2">Access permissions</a>
+
+To access your B2 bucket from CVAT, you need to create Application Keys:
```

Comment:
Presumably if your bucket is public, you don't need to do this? It would be useful to describe this.